Formula/mgis.rb

```diff
@@ -0,0 +1,42 @@
+class Mgis < Formula
+  desc "Provide tools to handle MFront generic interface behaviours"
+  homepage "https://thelfer.github.io/mgis/web/index.html"
+  url "https://github.com/thelfer/MFrontGenericInterfaceSupport/archive/refs/tags/MFrontGenericInterfaceSupport-2.1.tar.gz"
+  sha256 "f5b556aab130da0c423f395fe4c35d6bf509dd8fc958242f2e37ea788464aea9"
+  license any_of: ["LGPL-3.0-only", "CECILL-1.0"]
+  head "https://github.com/thelfer/MFrontGenericInterfaceSupport.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "llvm" => :build
+  depends_on "boost-python3"
+  depends_on "numpy"
+  depends_on "python@3.11"
+
+  def python3
+    which("python3.11")
+  end
+
+  def install
+    args = %W[
+      -Denable-portable-build=ON
+      -Denable-website=OFF
+      -Denable-enable-doxygen-doc=OFF
+      -Denable-c-bindings=ON
+      -Denable-fortran-bindings=ON
+      -Denable-python-bindings=ON # requires boost-python
+      -Denable-fenics-bindings=OFF # experimental and very limited
+      -Denable-julia-bindings=OFF # requires CxxWrap library
```

Comment:
In Ruby's `%W` word arrays, `#` is not regarded as the beginning of a comment.

```console
$ ruby -e "%W[
arg1 # comment 1
arg2 # comment 2
].each { |s| puts s }
"
arg1
#
comment
1
arg2
#
comment
2
```

So what happened was basically:

```console
$ brew install -sv mgis
[output truncated]
==> cmake -S . -B build -Denable-portable-build=ON -Denable-website=OFF -Denable-enable-doxygen-doc=OFF -Denable-c-bindings=ON -Denable-fortran-bindings=ON -Denable-python-bindings=ON # requires boost-python -Denable-fenics-bindings=OFF # experimental and very limited -Denable-julia-bindings=OFF # requires CxxWrap library -Denable-enable-static=OFF -Ddisable_python_library_linking=ON -DCMAKE_INSTALL_RPATH=$ORIGIN/../lib -DCMAKE_INSTALL_PREFIX=/home/linuxbrew/.linuxbrew/Cellar/mgis/2.1 -DCMAKE_INSTALL_LIBDIR=lib -DCMAKE_BUILD_TYPE=Release -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_VERBOSE_MAKEFILE=ON -Wno-dev -DBUILD_TESTING=OFF
CMake Warning:
  Ignoring extra path from command line:

   "library"

```

(I'm not sure how that worked, though. 🤷)

To add comments, you can use:

```ruby
args = [
  "arg1", # comment 1
  "arg2", # comment 2
]
```

Reply:
@maxence-wz Also, when you finish making changes, can you squash your commits into one and use the message `mgis 2.1 (new formula)`? Thanks!